draft-ietf-rats-corim.md

```diff
@@ -1107,6 +1110,16 @@ checks, and trust anchor checks.
 {::include cddl/crypto-key-type-choice.cddl}
 ~~~
 
+##### Integrity Registers {#sec-comid-integrity-registers}
+
+An Integrity register is a group of one or more named measurements.
+Each measurement has an unique identifier (either an unsigned integer or a string) and one or more associated digests.
+This type can be used to model the PCRs in a TPM or vTPM, in which case the identifier is the register index, as well as other vendor-specific "Integrity Registers".
```

Comment:
It is not clear from line 117 in the change whether we are using `This type` to refer to the identifier or Integrity register specifically. Hence I tried to word smith it to add better clarity, however feel free to improve it further, if need be!

Reply:
Somewhere in this section it might make sense to describe verifier behavior. I believe it is somewhat like ASN.1 SETS (vs. SEQUENCE) in that an entry "PCR5" at position 0 in the list should match a reference value even if the reference placed "PCR5" at position n>0. 

Maybe it goes without saying that verifier will not do type translation. That is if evidence has an entry `'5' : [...]` and reference values has an entry `5 : [...]` there is not ascii-to-numeric / numeric-to-ascii translation. What do you think?

Reply:
reworded in [0daa574](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/189/commits/0daa57453d485e1ad42dd2d338868d0b3d389fcd)